ui/fisher/static-data.js

```diff
@@ -9338,6 +9338,699 @@ let gFisherData = {
       '27514': '\u30ec\u30a4\u30f3\u30dc\u30fc\u30b7\u30e5\u30ea\u30f3\u30d7',
       '27515': '\u30d7\u30ec\u30c7\u30bf\u30fc\u30d5\u30a3\u30c3\u30b7\u30e5',
     },
+    'cn': {
+      '4925': '蝲蛄',
```

Comment:
Oh uhhhh since xivapi doesn't support cn so cn resources must be filled manually.
Maybe we can modify the python script to update the resources instead of establishing it from scratch? Thus we can leave cn resources there.
Moreover, there is a cn version of xivapi [here](https://github.com/thewakingsands/cafemaker/wiki), but we need to change the python script to auto-generate from that.

Reply:
I think I would prefer to have the script continue to just generate the file rather than update.

I could see either:
(1) have a separate javascript file for the cn additions and add them to the `static-data.js` data structures at runtime
(2) update the script to pull from the cn version of xivapi

Reply:
I prefer pulling from cn xivapi ;)

Reply:
Sounds good.  Sorry if I wasn't clear here, but I don't want to check in manual changes to this file.  So, if you want to land this pull request sooner, then maybe revert those changes, and can followup with script changes to pull from cn xivapi?

Reply:
reverted in https://github.com/quisquous/cactbot/pull/644/commits/9c6508cbff7c7168a8abb2479407de0f6eb2f40b
I'll update the script once cn xivapi updates the 5.0 fisher data